mm2src/coins/utxo/utxo_builder/utxo_coin_builder.rs

```diff
@@ -482,37 +480,44 @@ pub trait UtxoCoinBuilderCommonOps {
             );
         }
 
-        if args.negotiate_version {
-            event_handlers.push(ElectrumProtoVerifier { on_event_tx }.into_shared());
-        }
-
         let storage_ticker = self.ticker().replace('-', "_");
         let block_headers_storage = BlockHeaderStorage::new_from_ctx(self.ctx().clone(), storage_ticker)
             .map_to_mm(|e| UtxoCoinBuildError::Internal(e.to_string()))?;
         if !block_headers_storage.is_initialized_for().await? {
             block_headers_storage.init().await?;
         }
 
+        // Sort them by priority
         let mut rng = small_rng();
         servers.as_mut_slice().shuffle(&mut rng);
```

Comment:
The comment doesn't belong what happens its below

Reply:
Got rid of it